waves-nopeers.conf

```diff
@@ -0,0 +1,149 @@
+# Waves node settins
```

Comment:
There must be a better way to broadcast override only `known-peers`. What is this used for?